examples/documents/example.proto

```diff
@@ -28,88 +28,104 @@ message ExampleDocument {
     (proofs.hashed_field) = true
   ];
   Enum enum_type = 10;
+  repeated proofs.Salt salts = 11;
 }
 
 message AllFieldTypes {
   string string_value = 1;
   google.protobuf.Timestamp time_stamp_value = 2;
+  repeated proofs.Salt salts = 3;
 }
 
 message AllFieldTypesSalts {
   bytes string_value = 1;
   bytes time_stamp_value = 2;
+  repeated proofs.Salt salts = 3;
 }
 
 message SimpleItem {
   string valueA = 1;
+  repeated proofs.Salt salts = 2;
 }
 
 message RepeatedItem {
   repeated SimpleItem valueA = 1;
   string valueB = 2;
+  repeated proofs.Salt salts = 3;
 }
 
 message SimpleMap {
   map<int32,string> value = 1;
+  repeated proofs.Salt salts = 2;
```

Comment:
Is this needed here?

Reply:
It shouldn't needed it if it's only embedded in `NestedMap` or is it used somewhere else independently as well?

Reply:
if a message is only for embedded, it need not salts field